node/src/components/consensus/era_supervisor.rs

```diff
@@ -706,7 +707,10 @@ where
         era_id: EraId,
         proto_block: ProtoBlock,
     ) -> Effects<Event<I>> {
-        self.era_supervisor.metrics.time_since_proto_block.set(Timestamp::now().millis() as f64 / 1000.00);
+        self.era_supervisor
+            .metrics
+            .time_since_proto_block
```

Comment:
Should it be called `time_of_last_proto_block` then? Or maybe `time_of_last_proposed_block`… we'll probably get rid of the term "proto block" at some point.

Reply:
Can do!